### ISSUE TITLE
Adding proper code of conduct in the CODE_OF_CONDUCT.md

### ISSUE THREAD

**Issue by 1Shubham7** [oras-project/oras-www#212]
### Code of Conduct

- [X] I have read and agree to the ORAS' [Code of Conduct](https://github.com/oras-project/oras-www/blob/main/CODE_OF_CONDUCT.md)

### What part of document/web-page on oras.land is affected?

we can add proper code of conduct for contribution to the repo. This would be done by the project maintainers but I can create a basic blueprint of the md file

**Comment by TerryHowe** [comment]
I don't think we want to duplicate the CNCF code of conduct. I'm not sure how a change there would make things better.
